diff -urp ipw2200-1.1.4/Makefile b/Makefile
--- ipw2200-1.1.4/Makefile	2006-08-21 10:38:29.000000000 +0800
+++ b/Makefile	2006-08-21 17:02:26.000000000 +0800
@@ -185,6 +185,7 @@ patch_kernel:
 		${KSRC}/drivers/net/wireless/Makefile)
 	@cp README.ipw2200 ${KSRC}/Documentation/networking
 	@cp ipw2200.{c,h} ${KSRC}/drivers/net/wireless
+	@cp compat.{c,h} ${KSRC}/drivers/net/wireless
 	@cp in-tree/Kconfig.ipw2200 ${KSRC}/drivers/net/wireless
 	@(grep -q "Kconfig\.ipw2200" ${KSRC}/drivers/net/wireless/Kconfig || \
 	grep -q "IPW2200" ${KSRC}/drivers/net/wireless/Kconfig || \
