diff -urp ipw2200-1.1.4/ipw2200.c b/ipw2200.c
--- ipw2200-1.1.4/ipw2200.c	2006-08-21 10:38:32.000000000 +0800
+++ b/ipw2200.c	2006-08-21 17:02:02.000000000 +0800
@@ -82,9 +82,7 @@ MODULE_AUTHOR(DRV_COPYRIGHT);
 MODULE_LICENSE("GPL");
 
 static int cmdlog = 0;
-#ifdef CONFIG_IPW2200_DEBUG
 static int debug = 0;
-#endif
 static int channel = 0;
 static int mode = 0;
 
@@ -11973,10 +11971,8 @@ MODULE_PARM_DESC(auto_create, "auto crea
 module_param(led, int, 0444);
 MODULE_PARM_DESC(led, "enable led control on some systems (default 0 off)\n");
 
-#ifdef CONFIG_IPW2200_DEBUG
 module_param(debug, int, 0444);
 MODULE_PARM_DESC(debug, "debug output mask");
-#endif
 
 module_param(channel, int, 0444);
 MODULE_PARM_DESC(channel, "channel to limit associate to (default 0 [ANY])");
