--- ipw2200-1.2.0/Makefile.orig	2006-11-14 22:44:26.000000000 +0100
+++ ipw2200-1.2.0/Makefile	2006-11-14 22:46:17.000000000 +0100
@@ -30,14 +30,14 @@ CONFIG_IPW2200_MONITOR=y
 # simply uncomment:
 #
 # NOTE:  To use RADIOTAP you must also enable MONITOR above.
-#CONFIG_IPW2200_RADIOTAP=y
+CONFIG_IPW2200_RADIOTAP=y
 
 # The above monitor mode provides standard monitor mode.  The following
 # will create a new interface (named rtap%d) which will be sent all
 # 802.11 frames received on the interface
 #
 # NOTE:  To use PROMISCUOUS you must also enable MONITOR above.
-#CONFIG_IPW2200_PROMISCUOUS=y
+CONFIG_IPW2200_PROMISCUOUS=y
 
 endif
 
