--- ipw2200-1.2.2/ipw2200.c~	2008-03-02 22:02:22.000000000 +0100
+++ ipw2200-1.2.2/ipw2200.c	2008-03-02 22:33:37.100539785 +0100
@@ -11937,8 +11937,9 @@ static int ipw_pci_probe(struct pci_dev 
 		IPW_ERROR("Error allocating IRQ %d\n", pdev->irq);
 		goto out_destroy_workqueue;
 	}
-
+#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,24)
 	SET_MODULE_OWNER(net_dev);
+#endif
 	SET_NETDEV_DEV(net_dev, &pdev->dev);
 
 	mutex_lock(&priv->mutex);
