--- ipw2200-1.2.0/Makefile.orig	2006-10-16 12:34:38.000000000 +0200
+++ ipw2200-1.2.0/Makefile	2006-10-16 12:43:36.000000000 +0200
@@ -13,7 +13,12 @@
 # NOTE: If you have previously added the IPW project to your kernel 
 # 	and configured it for inclusion, these settings will be 
 #	overridden by your kernel configuration.
-ifndef CONFIG_IPW2200
+
+# NOTE: for PLD default kernels - we have got enabled as an module 
+# 	the standard IPW2200 driver (1.1.1 based) so CONFIG_IPW2200
+# 	is always set - we have to redefine config options here. 
+
+ifdef CONFIG_IPW2200
 EXTERNAL_BUILD=y
 CONFIG_IPW2200=m
 CONFIG_IPW2200_DEBUG=y
@@ -30,14 +35,14 @@
 # simply uncomment:
 #
 # NOTE:  To use RADIOTAP you must also enable MONITOR above.
-#CONFIG_IPW2200_RADIOTAP=y
+CONFIG_IPW2200_RADIOTAP=y
 
 # The above monitor mode provides standard monitor mode.  The following
 # will create a new interface (named rtap%d) which will be sent all
 # 802.11 frames received on the interface
 #
 # NOTE:  To use PROMISCUOUS you must also enable MONITOR above.
-#CONFIG_IPW2200_PROMISCUOUS=y
+CONFIG_IPW2200_PROMISCUOUS=y
 
 endif
 
